infra/templates/az-isolated-service-single-region/terraform.tfvars

```diff
@@ -0,0 +1,9 @@
+# GENERAL
+resource_group_location = "eastus2"
```

Comment:
Can we confirm if this region is the same that our target customer is using in their ASE envrionment?

Reply:
The deployment will fail if they don't match so this validation is done server side for us.

There is no data resource available for us to load this up natively so I'm ok leveraging server side validation